save-agent/src/linuxX64Main/kotlin/com/saveourtool/save/agent/SaveAgent.kt

```diff
@@ -192,8 +192,10 @@ class SaveAgent(internal val config: AgentConfiguration,
                         testResultStatus,
                         executionStartSeconds.value,
                         currentTime.epochSeconds,
-                        missingWarnings = debugInfo.debugInfo?.countWarnings?.missing,
-                        matchedWarnings = debugInfo.debugInfo?.countWarnings?.match,
+                        unmatched = debugInfo.debugInfo?.countWarnings?.missing?.toLong() ?: 0L,
```

Comment:
debugInfo is not available (by some reason) -- checks counts are set to `0` instead of a null (which marks that value is not set yet)